### ISSUE TITLE
Add FAQ for use image in podman

### ISSUE THREAD

**Issue by 6PATyCb** [jgiannuzzi/docker-gitolite#16]
If someone is using podman to run this image, don't forget to add the <code>exec</code> option when mounting the <code>gitolite-git</code> volume like this:

`podman run -d --privileged --name gitolite -p 2200:22 -v gitolite-sshkeys:/etc/ssh/keys -v gitolite-git:/var/lib/git:exec jgiannuzzi/gitolite`

Without this, all hooks will not work.

**Comment by hydrargyrum** [comment]
why the `--privileged`?

**Comment by 6PATyCb** [comment]
Good point. I can't answer that right now. Maybe that flag got accidentally added to the command while I was trying to run it.
